### DISCUSSION TITLE
How to get all radio groups from a page ?

### DISCUSSION THREAD

**Discussion by AkshayJangid** [cloudscape-design/components#1811] [Q&A] [answered]
Hey Team, 
I have multiple radio groups on a page. 
I'm using for testing @cloudscape-design/components/test-utils/dom and i want to find all radio groups of a page. 
What i have tried is : 
const wrapper = createWrapper(container);
    const radioGroups = wrapper.findRadioGroup;

This returns only the first radio group. How do i find all radio groups ?

**Comment by Al-Dani** [comment, marked as answer]
Hey,
We do not provide a way to find an array of all specific components. What we suggest is to find required component using a selector.
You can assign a `data-testid` attribute to a radioGroup and use it as a selector in `.findRadioGroup('[data-testid=radioGroup1]')`.

Hope that helps!
